src/utility/onHasCompletion.js

```diff
@@ -82,7 +90,7 @@ export default function onHasCompletion(cm, data, onHintInformationRender) {
 
     // Now that the UI has been set up, add info to information.
     const description = ctx.description ?
-      marked(ctx.description, { smartypants: true }) :
+      marked(ctx.description, { sanitize: true }) :
```

Comment:
Just making this consistent with everywhere else we use marked, in my experience smartypants has too many false-positive conversions which can look worse, and especially isn't great for things other than long-form content. sanitize is important to add though :)